db/bigtable_eth1.go

```diff
@@ -1252,6 +1280,11 @@ func (bigtable *Bigtable) TransformItx(blk *types.Eth1Block, cache *freecache.Ca
 	bulkData = &types.BulkMutations{}
 	bulkMetadataUpdates = &types.BulkMutations{}
 
+	ContractUpdateWrites := &types.BulkMutations{
+		Keys: make([]string, 0),
+		Muts: make([]*gcp_bigtable.Mutation, 0),
+	}
```

Comment:
**Question:** Why didn't you use `bulkData` for that? By NOT using it, the `ContractUpdateWrites` keys are not stored via `SaveBlockKeys`. Is that intentional?

Reply:
Oh yeah that's correct.. I forgot about re-orgs, my bad, thank you :sweat_smile: I'll take care of that